src/collections/Orders/index.ts

```
@@ -0,0 +1,221 @@
+import { admin } from '@/access/admin'
+import { user } from '@/access/user'
+import { CollectionConfig } from 'payload'
+import { generateOrderNumber } from './hooks/generateOrderNumber'
+import { Effect, Console, pipe, HashMap, Option, Array, HashSet } from 'effect'
```

Biome (1.9.4)

[error] Do not shadow the global "Array" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)